### ISSUE TITLE
Repetitive notebook screen reader message

### ISSUE THREAD

**Issue by eleanorjboyd** [microsoft/vscode#211131]
Was working on this TPI: https://github.com/microsoft/vscode/issues/210661

And found that I got a repeat message in the notebook a lot which got annoying to hear over and over again as I switched tabs then returned to the notebook. If this is configurable then probably already solved but would be good to reduce how much duplicate explanation is done. Here is the specific wording that kept coming up:
![Screenshot 2024-04-23 at 9 59 47 AM](https://github.com/microsoft/vscode/assets/26030610/8dbbc51a-4576-4344-bddb-e9e838852ac2)

**Comment by meganrogge** [comment]
thanks for the issue. that's coming from the screen reader itself and we cannot change it. SR users are used to dismissing repetitive info like this
